zio-http/jvm/src/test/scala/zio/http/ZClientAspectSpec.scala

```diff
@@ -100,6 +103,57 @@ object ZClientAspectSpec extends ZIOHttpSpec {
           extractStatus(response) == Status.Ok,
         ),
       ),
+      suite("curl request logger")(
```

Comment:
Can you add one with a streaming body to check that it is not logged?

Reply:
That's a good idea! I've just added a new commit that implements a test for the logging of a streaming request body.